README.md

```diff
@@ -11,9 +11,21 @@ ScalarDB is a universal HTAP engine. It achieves ACID transactions and real-time
   <a href='https://www.youtube.com/watch?v=lW6nCEJV-gs' target="_blank"><img src='docs/images/scalardb-video-preview.png' width='70%'></a>
 </p>
 
-## Docs
+To learn more, see [ScalarDB Overview](https://scalardb.scalar-labs.com/docs/latest/overview).
 
-See our [User Documentation](https://scalardb.scalar-labs.com/docs/latest/).
+## Quickstart
+
+1. Start running transactions.
+   - **Get started with ScalarDB Core:** To set up a basic application by using Java, see [Get Started with ScalarDB](https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb).
+   - **Get started with ScalarDB Cluster (for enterprise customers):** To set up a basic application that uses ScalarDB Cluster through the Java API, see [Get Started with ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/getting-started-with-scalardb-cluster).
```

Comment:
```suggestion
   - **Get started with ScalarDB Cluster (for enterprise customers):** To set up a basic application that uses ScalarDB Cluster through the Java API, see [Getting Started with ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/getting-started-with-scalardb-cluster).
```
Ditto.

Reply:
As I mentioned in my previous comment reply, let's use the current title here and update it later when we change the title of the doc.